### ISSUE TITLE
Bind ⌘R to Send button

### ISSUE THREAD

**Issue by ctessmer** [httpie/desktop#36]
# Checklist

- [x] I've searched for similar feature requests.

# Enhancement request

Bind ⌘R to `Send` button instead of `Reload`.

# Problem it solves

## TL;DR
Pressing ⌘R refreshes the contents of the Httpie desktop window—instead of resending the request.

## Details

This sequence is muscle memory for web developers:

1. Update code in editor
2. Save via `⌘S` (cmd + s]
3. Switch to browser via `⌘Tab` [cmd + tab]
4. Refresh page (aka: resend request) via `⌘R` (cmd + r]
5. Inspect changes

Rinse & repeat.

Now replace the browser with Httpie desktop. 

Same muscle memory, different outcome. Now, when pressing `⌘R` in step 4, the whole contents of the Httpie app window updates instead of just resending the request as is intended. 

Currently we're forced to reposition the mouse over the `Send` button and click each time to resend the current request.

I mean… how are we even supposed to live?! 🤣

**Comment by wlkns** [comment]
Bump. This issue is also affecting me and another. You naturally expect `⌘R` to resend/refresh the request but instead the entire window reloads.

**Comment by wlkns** [comment]
@ctessmer I've got an interim solution for you and perhaps others with this issue...  muscle memory means it became a repeated issue for me and I was about to ditch Httpie...

Toggle developer tools (View menu at top)

In the console, enter the following —

```js
window.addEventListener("keydown", (e) => {
  if ( ["s", "r"].includes(e.key) && e.metaKey ) {
    e.preventDefault();
    document.querySelector('[data-testid="url-bar-submit"]').click()
  }
})
```

This will intercept and Cmd+S or Cmd+R and click the send button, so should be compatible with updates etc.

**Comment by jkbrzt** [comment]
🚢 Shipped in [v2024.1.1 →](https://github.com/httpie/desktop/releases/tag/)
